hackathon_site/hardware/views.py

```diff
@@ -17,11 +19,18 @@
 
 logger = logging.getLogger(__name__)
 
+from hardware.api_filters import HardwareFilter
```

Comment:
Move this up to be near the rest of the imports (above the `from hardware.models ...` import)